en/development/debugging.rst

```diff
@@ -51,9 +51,13 @@ Using the Debugger Class
 
 .. php:class:: Debugger
 
+
 To use the debugger, first ensure that ``Configure::read('debug')`` is
 set to ``true``.
 
+You can use ``filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` in **config/app.php** file to ensure that ``debug`` is a boolean.
```

Comment:
@markstory I dont think the sentence was meant like this, or should be merged as such. In the scope of this topic it doesn't bear any useful info.

Reply:
Refs my clear objection @ https://github.com/cakephp/docs/pull/6326#pullrequestreview-332268676